src/taskpane/taskpane.ts

```diff
@@ -37,3 +42,65 @@ async function run() {
     OfficeHelpers.Utilities.log(error);
   }
 }
+
+async function runCfTests() {  
+  await Excel.run(async context => {
+    for (let i = 0; i < testFunctions.length; i++) {
+      const range = context.workbook.getSelectedRange();
+      const formula : string = testFunctions[i]
+      range.formulas = [[formula]];
+      await context.sync();
+      await sleep(2000);
+
+      // Check to if this is a streaming function
+      await readData(formula.indexOf("INCREMENT") > 0)
+    }
+  });
+  sendData(cfValues);
+}
+
+async function readData(isStreamingFunction: boolean) {
+  await Excel.run(async context => {
+
+    // if this is a streaming function, we want to capture two values so we can
+    // validate the function is indeed streaming
+    for (let i = 0; isStreamingFunction ?  i <= 1 : i < 1; i++)
```

Comment:
It would be simpler to define as `readData(count: number = 1)` where count is > 1 for a streaming function, and the for loop just counts down to zero.